test/application.test.js

```diff
@@ -326,6 +326,20 @@ describe('Feathers application', () => {
     assert.equal(otherApp.mixins.length, 4);
   });
 
+  it('setup only services work (#285)', done => {
```

Comment:
I don't really know what this means? Maybe we can rephrase to "initializes a service with only a setup method"?